src/custom/state/appData/updater.ts

```diff
@@ -78,8 +77,31 @@ async function _uploadToIpfs(
       }
     } catch (e) {
       // TODO: add sentry error to track soft failure
-      console.debug(`[UploadToIpfsUpdater] Failed to upload doc, will try again`, JSON.stringify(doc))
-      updatePending({ chainId, orderId, uploading: false, tryAfter: Date.now() + TRY_AGAIN_AFTER })
+      console.debug(
```

Comment:
not this PR but add sentry error to track hard failure Doesn't it already track it every time we write in the console.error? (we use console.error above and  we have a similar TODO)

Also, why DEBUG??

Reply:
> Doesn't it already track it every time we write in the console.error?

Dunno. Does it? I thought we needed to wrap it the helper function to explicitly and nicely track things there

cc @W3stside ?

> If it's because it will be re-attempted, then at least should be warn, right?

Sure, will make it a `warn`

Reply:
I've just checked, and is not capturing by default. 

Up to us if we want to capture it. Maybe yes? 

https://stackoverflow.com/questions/50633580/how-to-report-console-error-with-sentry